File: data-quality/trash/DisposableCheckerTest.scala
package it.gov.daf.dataquality.analysis.email.checkers

import org.scalatest.FunSuite

/**
  * @author Marco Bianchi (bianchi74@gmail.com)
  */
class DisposableCheckerTest extends FunSuite {

  test("testIsChecked") {

    val checker = DisposableChecker.getInstance();
    assert(checker.isChecked("username@despam.it"))
    assert(! checker.isChecked("username@notDisposable.it"))
  }

}

File: data-quality/src/test/java/it/gov/daf/dataquality/analysis/email/checkers/DisposableCheckerTest.java
package it.gov.daf.dataquality.analysis.email.checkers;

import junit.framework.TestCase;
import org.junit.Test;

/**
 * @author Marco Bianchi (bianchi74@gmail.com)
 */
public class DisposableCheckerTest extends TestCase {

    @Test
    public void testIsChecked(){
        Checker checker = DisposableChecker.getInstance();
        assert(checker.isChecked("username@despam.it"));
        assert(! checker.isChecked("username@notDisposable.it"));
    }
}
